doc/man3/SSL_get_version.pod

```diff
@@ -19,12 +19,14 @@ protocol information of a connection
 
 =head1 DESCRIPTION
 
-SSL_client_version() returns the protocol version used by the client when
-initiating the connection. SSL_get_version() returns the name of the protocol
-used for the connection. SSL_version() returns the protocol version used for the
-connection. They should only be called after the initial handshake has been
-completed. Prior to that the results returned from these functions may be
-unreliable.
+SSL_client_version() returns the protocol version advertised by the client in
+the legacy_version field of the ClientHello when initiating the connection.
+Note that, for TLS, this value will never indicate a version greater than
+TLSv1.2 even if TLSv1.3 is subsequently negotiated. SSL_get_version() returns
+the name of the protocol used for the connection. SSL_version() returns the
+protocol version used for the connection. They should only be called after the
```

Comment:
I would make that "numeric protocol version" to clarify exactly what it is.  Yes, I know that one can infer that from the return value, but still, I think that from just reading the text, such clarification is needed to make a better distinction between this and `SSL_get_version`.